pkgs/applications/audio/qpwgraph/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, mkDerivation, fetchFromGitLab
+, cmake, pkg-config
+, alsa-lib, pipewire
+, ...
```

Comment:
```suggestion
```

Reply:
Thanks for catching this! Forgot that `callPackage` works like that.